docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Update functions for hs_paragraph_types module.
+ */
+
+use Drupal\Core\Site\Settings;
+use Drupal\Core\Config\FileStorage;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Store image gallery display mode in paragraph variable.
+ */
+function hs_paragraph_types_update_9401() {
+  // Load field_photoalbum_styl field definition exported configuration.
+  $config_directory = new FileStorage(Settings::get('config_sync_directory'));
+  $field_storage = $config_directory->read('field.storage.paragraph.field_photoalbum_style');
+  $field_config = $config_directory->read('field.field.paragraph.stanford_gallery.field_photoalbum_style');
+
+  // Create field_photoalbum_style from exported configuration.
+  if (!FieldStorageConfig::loadByName($field_storage['entity_type'], $field_storage['field_name'])) {
+    FieldStorageConfig::create($field_storage)->save();
+  }
+  if (!FieldConfig::loadByName($field_config['entity_type'], $field_config['bundle'], $field_config['field_name'])) {
+    FieldConfig::create($field_config)->save();
+  }
+
+  // Copy 'display_mode' behavior value to new field in stanford_gallery.
+  $paragraphs = \Drupal::entityTypeManager()
+    ->getStorage('paragraph')
+    ->loadByProperties(['type' => 'stanford_gallery']);
+  foreach ($paragraphs as $paragraph) {
+    $paragraph->field_photoalbum_style->value = $paragraph->getBehaviorSetting('image_gallery', 'display_mode');
```

Comment:
use `set` methods..

Why move the behavior settings instead of just using the behavior itself?

Reply:
@pookmish See my previous comment on the decision to not use a behavior. The following components were using a configurable field implementation of `field_type: display_mode_field` - Given this was used for more existing paragraphs than the behavior implementation we opted change this one paragraph for consistency. If you strongly prefer the behavior implementation we can create a tech-debt ticket to convert this paragraph back to a behavior and change the implementation for the others. 
- callout box
- color band
- postcard
- spotlight

Reply:
yes, let's create a ticket to move style related fields into behaviors. It will be more consistent with other products my team maintains.

Reply:
@pookmish We have a ticket created for Tori/Amit to prioritize (https://app.clickup.com/t/36718269/SHS-5004) . Is this current implementation a merge blocker?

Reply:
No. we can work on that later.